### DISCUSSION TITLE
Blender Gis Height map Problem

### DISCUSSION THREAD

**Discussion by Dprestino** [domlysz/BlenderGIS#614] [General]
Hi everyone,
It seems that Blender Gis has not downloaded the altitude maps from google for a couple of days, as if the servers are no longer connected, how do we solve?
Thank you!

**Comment by TimEBN2020** [comment]
I am having a similar problem. Specifically, when I try to collect OpenTopography SRTM 30m data, I get the error "Cannot reach OpenTopography web service. check logs for more infos". Please help! My logfile is attached.

[bgis.log](https://github.com/domlysz/BlenderGIS/files/8100983/bgis.log)

**Reply by Leandro77Tinoco** [reply]
Sigo reportando error con respecto al SRTM 30m data!
Por favor ayuda para solucionarlo.
